database-mysqldb/Accounts.js

```diff
@@ -8,15 +8,9 @@ const Account = db.define('account', {
     unique: true,
     allowNull: false,
   },
-  buying_power: {
-    type: Sequelize.DECIMAL(14, 4),
-  },
-  option_level: {
-    type: Sequelize.INTEGER,
-  },
-  watchlist: {
-    type: Sequelize.STRING,
-  },
+  buying_power: Sequelize.DECIMAL(14, 4),
+  option_level: Sequelize.INTEGER,
+  watchlist: Sequelize.STRING,
```

Comment:
If we refer back to your original seed for Accounts.js, maybe we can consider generating the 100 stocks dynamically, versus hard-coding it?

Reply:
Also, in seed.js again, why did you use different types to represent buying_power vs quantity? (and choosing to use underscores in certain variable names over others). Are you trying to address floating point precision?

Reply:
The Sequelize.DECIMAL(14, 4) vs. the Sequelize.INTEGER made me take a second look

Reply:
Yep good catch - DECIMAL is specifically used here to address the floating point precision issue.